src/server/GameServer.ts

```diff
@@ -41,6 +41,8 @@ export class GameServer {
   private turns: Turn[] = [];
   private intents: Intent[] = [];
   public activeClients: Client[] = [];
+  // Used to track the Creator of a Private Lobby (to kick players)
+  private privateLobbyCreatorID: string | null = null;
```

Comment:
nit: use `undefined` instead of `null` here
```suggestion
  private privateLobbyCreatorID: string | undefined;
```

Reply:
Implemented